airflow/www/templates/airflow/dag.html

```diff
@@ -22,6 +22,9 @@
 {% block head_css %}
   {{ super() }}
   <link href="{{ url_for_asset('bootstrap-toggle.min.css') }}" rel="stylesheet" type="text/css">
+  <meta name="dag-id" content="{{ dag.dag_id }}">
```

Comment:
For  consistency it probably makes sense to have these in meta, but having `<html data-dag-id="...">` (or on some other element in the body) would be "semantically correct" for me on this page, as the DAG ID is a property of the document/content.

But I'm also fine with it staying as a meta tag.

Reply:
It would be a bit harder to add an additional attribute to html. So I will leave the current version not to add additional complexity.